bin/test_module

```diff
@@ -102,6 +120,14 @@ if [ -n "$failed" ]; then
     false
 fi
 
+#now interrupt the process.  get its PID:
+echo Stopping module pcap...
+pid=$(ps -e | pgrep tcpdump)  
```

Comment:
use $! from when after the command is executed, rather than trying to extract the PID.

Reply:
Not sure what is being asked here.  Extraction of PID highlighted but indicating changing the command in some manner section to remove this necessity.

Reply:
https://stackoverflow.com/questions/1908610/how-to-get-process-id-of-background-process
You want to have `pid=$!` (at the appropriate point in the file), rather than greping through ps

Reply:
Got it, will implement.